### ISSUE TITLE
cover images should have sensible alt text

### ISSUE THREAD

**Issue by bkalish** [forbeslibrary/wordpress.plugin.staff-picks#8]
Currently, if the user does not set `alt` text for an image, WordPress uses the filename of the upload. Either the user should be forced to set `alt` text when uploading the image, or the plugin should detect blank alt text and provide something sensible in its place.
